packages/shared/src/components/squads/Comment.tsx

```diff
@@ -203,7 +207,12 @@ export function SquadComment({
               className="btn-primary-cabbage"
               type="submit"
               loading={isLoading}
-              disabled={!commentary || isLoading || isCheckingUrl}
+              disabled={
+                !commentary ||
+                isLoading ||
+                isPreviewLoading ||
+                (!form?.post?.post && !form?.externalLink?.title)
+              }
```

Comment:
Nitpicking, but maybe move this to a const above just for clarity

Reply:
Agree